tests/test_octodns_record_ds.py

```diff
@@ -12,64 +12,64 @@
 class TestRecordDs(TestCase):
     def test_ds(self):
         for a, b in (
-            # diff flags
+            # diff key_tag
             (
                 {
-                    'flags': 0,
-                    'protocol': 1,
-                    'algorithm': 2,
-                    'public_key': 'abcdef0123456',
+                    'key_tag': 0,
+                    'algorithm': 1,
+                    'digest_type': 2,
+                    'digest': 'abcdef0123456',
```

Comment:
We'll want to leave the original key names in at least one of these.

Reply:
Sure and to reach 100% test coverage I needed a bunch of them :)